code/modules/client/loadout/loadout_suit.dm

```diff
@@ -0,0 +1,87 @@
+/datum/gear/suit
+	subtype_path = /datum/gear/suit
+	slot = slot_wear_suit
+	cost = 2
+	sort_category = "External Wear"
+
+//WINTER COATS
+/datum/gear/suit/coat
+	subtype_path = /datum/gear/suit/coat
+
+/datum/gear/suit/coat/grey
+	display_name = "winter coat"
+	path = /obj/item/clothing/suit/hooded/wintercoat
+
+/datum/gear/suit/coat/job
+	subtype_path = /datum/gear/suit/coat/job
+	subtype_cost_overlap = FALSE
+
+/datum/gear/suit/coat/job/sec
+	display_name = "winter coat, security"
+	path = /obj/item/clothing/suit/hooded/wintercoat/security
+	allowed_roles = list("Head of Security", "Master-at-Arms", "Detective", "Security Officer", "Security Pod Pilot")
+
+/datum/gear/suit/coat/job/captain
+	display_name = "winter coat, captain"
+	path = /obj/item/clothing/suit/hooded/wintercoat/captain
+	allowed_roles = list("Captain")
+
+/datum/gear/suit/coat/job/med
+	display_name = "winter coat, medical"
+	path = /obj/item/clothing/suit/hooded/wintercoat/medical
+	allowed_roles = list("Chief Medical Officer", "Medical Doctor", "Chemist", "Psychiatrist", "Paramedic", "Virologist", "Brig Physician")
+
+/datum/gear/suit/coat/job/sci
+	display_name = "winter coat, science"
+	path = /obj/item/clothing/suit/hooded/wintercoat/science
+	allowed_roles = list("Scientist", "Research Director")
+
+/datum/gear/suit/coat/job/engi
+	display_name = "winter coat, engineering"
+	path = /obj/item/clothing/suit/hooded/wintercoat/engineering
+	allowed_roles = list("Chief Engineer", "Engineer", "Mechanic")
```

Comment:
Loadouts support alt job titles. I plan on fixing alt job titles soon, so I left Mechanic and other alt titles in.

Let me know if you want to remove them, though I'll just be adding them again soon.